src/functions/functionsDialog.component.tsx

```diff
@@ -268,16 +291,54 @@ const FunctionsDialog = (props: FunctionsDialogProps) => {
       label: func.name,
     }));
 
+  React.useEffect(() => {
+    setSelectedColIds(
+      appliedFunctions
+        .filter((func) => appliedSelectedIds.includes(func.name))
+        .map((func) => func.id)
+    );
+  }, [appliedFunctions, appliedSelectedIds]);
+
   return (
-    <Dialog open={open} onClose={onClose} maxWidth="lg" fullWidth>
+    <Dialog open={open} onClose={onClose} maxWidth="xl" fullWidth>
       <DialogTitle>Functions</DialogTitle>
       <DialogContent>
         <Grid container columnSpacing={2}>
           <Grid container item xs pr={1} flexDirection="column" rowSpacing={1}>
             <Heading mt={1}>Enter function</Heading>
             {functions.map((func, index) => {
               return (
-                <Grid container item key={func.id}>
+                <Grid pl={0} container item key={func.id}>
+                  <Grid item>
+                    <Tooltip
+                      title={
+                        selectedColIds.includes(func.id)
+                          ? 'Removed function column'
```

Comment:
`Removed` -> `Remove function column` (or maybe even `Hide` to contrast better with `Display`?)